bucket/theia-ide.json

```
@@ -0,0 +1,39 @@
+{
+    "version": "1.49.100",
+    "description": "A modern and open IDE for cloud and desktop. Theia platform based. Formerly “Theia Blueprint”.",
+    "homepage": "https://theia-ide.org/#theiaide",
+    "license": "EPL-2.0, GPL-2.0, MIT",
+    "notes": "Settings are stored in '%APPDATA%\\Theia IDE', and are not persisted by Scoop.",
+    "architecture": {
+        "64bit": {
+            "url": "https://download.eclipse.org/theia/ide/1.49.100/windows/TheiaIDESetup-1.49.100.exe#/dl.7z",
+            "hash": "sha512:a88a979fb1446cf463517377331ed67c78cdb9e2dec70e5d139b1b751883e2cecb20d45f060d9a1a8b30d7fb374d98ddd089e60c071cb5d8fd35819ba06a9156"
+        }
+    },
+    "extract_dir": "$PLUGINSDIR",
+    "installer": {
+        "script": "Expand-7zipArchive \"$dir\\app-64.7z\" \"$dir\" -Removal"
+    },
+    "bin": "TheiaIDE.exe",
+    "shortcuts": [
+        [
+            "TheiaIDE.exe",
+            "TheiaIDE"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/eclipse-theia/theia",
+        "replace": "${1}00"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://download.eclipse.org/theia/ide/$version/windows/TheiaIDESetup-$version.exe#/dl.7z",
```

```suggestion
                "url": "https://www.eclipse.org/downloads/download.php?file=/theia/ide/$version/windows/TheiaIDESetup-$version.exe&r=1#/dl.7z",
```

* this download link is (inspired from) official download link on theia webpage
* it resolves the mirror itself
* remove the `&r=1` to view the download page & select the mirror manually


for me, the link resolved was this following one, which was excruciatingly slow when used with `checkhashes.ps1`: 

```xml
<mirror url="https://mirror.kakao.com/eclipse/theia/ide/latest/windows/TheiaIDESetup-1.49.100.exe" label="[Korea, Republic Of] Kakao Corp. (https)"/>
```

but the one i used above works super fast. so, i ain't sure if the automatic mirror resolution should be used or not...

```url
https://download.eclipse.org/theia/ide/latest/windows/TheiaIDESetup-1.49.100.exe
```

it is the landing/redirected location from the link listed on downloads page under the `mirror_id=1` i.e. `Main Download Site` `Canada - [Main eclipse.org downloads area]` (following link):

```url
https://www.eclipse.org/downloads/download.php?file=/theia/ide/latest/windows/TheiaIDESetup-1.49.100.exe&mirror_id=1
```